craft_application/commands/lifecycle.py

```diff
@@ -482,10 +482,6 @@ def _run(
 
         if parsed_args.test_path:
             testing_service.validate_tests(parsed_args.test_path)
-        # Output into the spread directory.
-        parsed_args.output = pathlib.Path.cwd() / "spread"
-        parsed_args.output.mkdir(exist_ok=True)
-        parsed_args.fetch_service_policy = None
```

Comment:
what about this fetch service policy change?

Reply:
@lengau Why was this change needed when changing the output directory?

Reply:
The policy change was unrelated - we were just setting it so when the `pack` command checks for it, it would still be set.